### PR TITLE
2768 Add parameter bias for conv blocks

### LINKED ISSUE #2768: Enhance some networks to set `bias=False` for conv layers
If a conv layer is directly followed by a batch norm layer, the bias in the conv layer is not need, because the `subtract the mean` manipulation will cancel out the effect of bias.

[Reference:](https://pytorch.org/tutorials/recipes/recipes/tuning_guide.html)

In MONAI's `Convolution` class, the default value of `bias` is `True`, and the following networks/blocks cannot configure the `bias` parameter when using `Convolution`:

- `SimpleASPP`
- `HighResNet`
- `VNet`
- `UNet`
- `BasicUNet`
- `AutoEncoder`

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Signed-off-by: Yiheng Wang <vennw@nvidia.com>

Fixes #2768 .

### Description
Except for add parameter bias, this PR also changes `UpCat` to support optional `x_e` in forward function to support encoders that have different number of feature maps .

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by wyli — APPROVED** [review]
Thanks!
